polymorphic/query.py

```diff
@@ -25,6 +25,25 @@
 Polymorphic_QuerySet_objects_per_request = CHUNK_SIZE
 
 
+if django.VERSION >= (1, 9):
+    # We ignore this on django < 1.9, as ModelIterable didn't yet exist.
+    from django.db.models.query import ModelIterable
+
+
+    class PolymorphicModelIterable(ModelIterable):
+        def __iter__(self):
+            base_iter = super(PolymorphicModelIterable, self).__iter__()
+
+            if self.queryset.polymorphic_disabled:
+                for o in base_iter:
+                    yield o
+                return
+
+            real_instances = self.queryset._get_real_instances(base_iter)
+            for obj in real_instances:
+                yield obj
```

Comment:
Should we be chunking with `Polymorphic_QuerySet_objects_per_request`?

Django 1.11 introduced [server-side cursors](https://docs.djangoproject.com/en/1.11/releases/1.11/#database-backends) for postgres... I have no idea how this would play with that.